example/index.html

```diff
@@ -1,12 +1,13 @@
 <html>
   <head>
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
-    <title>Faster Re-Load</title>
+    <title>Hello Ines</title>
```

Comment:
Indeed! We could totes have a much nicer **example/index.html**
Want to contribute one? :-)